e2e/passkey/passkeyProxyV2.ethereum.test.ts

```diff
@@ -46,21 +42,56 @@ describe('Passkey Pallet Proxy V2 Ethereum Tests', function () {
         false
       );
       const passkeyProxy = ExtrinsicHelper.executePassKeyProxyV2(fundedSr25519Keys, passkeyPayload);
-      await assert.doesNotReject(passkeyProxy.fundAndSendUnsigned(fundingSource));
-      await ExtrinsicHelper.waitForFinalization((await getBlockNumber()) + 2);
-      // adding some delay before fetching the nonce to ensure it is updated
-      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
We don't need to await finalization now that `sendUnsigned` has been fixed to not resolve until the extrinsic is included in a finalized block.